src/hypothesis/strategies.py

```diff
@@ -910,6 +912,108 @@ def do_draw(self, data):
     return PermutationStrategy()
 
 
+@defines_strategy
+def datetimes(min_datetime=dt.datetime.min, max_datetime=dt.datetime.max,
+              timezones=None):
```

Comment:
Sure.  I was trying to allow a `just(min_datetime)` case, but consistency of API is probably better than implementation details.